core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -606,6 +606,16 @@ public void resetState() {
     mPathProperties.resetState();
   }
 
+  @Override
+  public void updateConfig(Map<String, String> propertiesMap) {
+    for (Map.Entry<String, String> entry : propertiesMap.entrySet()) {
+      PropertyKey propertyKey = PropertyKey.fromString(entry.getKey());
+      if (propertyKey.isDynamic()) {
+        ServerConfiguration.set(propertyKey, entry.getValue());
```

Comment:
nit: i know this is equivalent, but let's be a bit more explicit here
```suggestion
        ServerConfiguration.set(propertyKey, entry.getValue(), Source.RUNTIME);
```

Reply:
Done